### ISSUE TITLE
SSL error in data received

### ISSUE THREAD

**Issue by almereyda** [matrix-org/pantalaimon#10]
Sometimes an ssl.SSLError appears:

```
SSL error in data received
protocol: <asyncio.sslproto.SSLProtocol object at 0x7f1b099463c8>
transport: <_SelectorSocketTransport fd=199 read=polling write=<idle, bufsize=0>>
Traceback (most recent call last):
  File "/usr/lib64/python3.7/asyncio/sslproto.py", line 526, in data_received
    ssldata, appdata = self._sslpipe.feed_ssldata(data)
  File "/usr/lib64/python3.7/asyncio/sslproto.py", line 207, in feed_ssldata
    self._sslobj.unwrap()
  File "/usr/lib64/python3.7/ssl.py", line 767, in unwrap
    return self._sslobj.shutdown()
ssl.SSLError: [SSL: KRB5_S_INIT] application data after close notify (_ssl.c:2609)
```

This occured when using Fractal.

**Comment by almereyda** [comment]
This has also been seen when using Quaternion.

**Comment by poljar** [comment]
This is an upstream bug by aiohttp or even further up the stream by one of its dependencies https://github.com/aio-libs/aiohttp/issues/3535.

**Comment by poljar** [comment]
It seems that SSL errors getting to the stderr are an underlying python problem and the issue will be fixed in python 3.8, source https://github.com/aio-libs/aiohttp/issues/3675#issuecomment-478595338 and https://github.com/aio-libs/aiohttp/issues/3292#issuecomment-424696255.

**Comment by poljar** [comment]
I believe this is fixed in python 3.8 now.
